src/Uno.UI/UI/Xaml/FontFamily.cs

```diff
@@ -16,6 +16,7 @@ public FontFamily(string familyName)
 
 			// A workaround before font fallback is supported. Issue: https://github.com/unoplatform/uno/issues/10148 
 			if (familyName.Contains("Segoe Fluent Icons", StringComparison.InvariantCultureIgnoreCase) ||
+				familyName.Contains("Segoe UI Symbol", StringComparison.InvariantCultureIgnoreCase) ||
```

Comment:
Where is this one used? I did not find it inside our own repo, maybe third party libraries?

Reply:
Existing WinUI XAML code can use this explicitly, which would then render invalid characters on our side. WinUI has hopefully updated all occurrences to `{ThemeResource}`, but users can still hit it. We already had fallbacks for the other icon fonts so this is mainly for completeness

Reply:
@MartinZikmund To me this doesn't look right. This was a workaround before we had font fallback. Right now we will fall back to what SkiaSharp.SKFontManager.Default.MatchCharacter returns. What we would need to do is to special case those 4 fonts so that they fall back to the symbols font specifically (if available), but only if they don't exist (i.e. if they're not provided by the system).